lib/ast-converter.js

```diff
@@ -1048,6 +1048,7 @@ module.exports = function(ast, extra) {
             case SyntaxKind.GetAccessor:
             case SyntaxKind.SetAccessor:
             case SyntaxKind.MethodDeclaration:
+                var offset = node.parameters.pos - node.name.end - 1;
```

Comment:
This calculation ends up containing both `+node.name.end` and `-node.name.end` so we could simplify the range start to just be `node.parameters.pos - 1`, and scrap this variable altogether right?

As an aside, most of the conversion currently happens within one single function scope right now, so please give try and give variables more specific and unique names. I will be refactoring at some point once the project has fully stabilised, but just something to bear in mind at the moment.